lib/hammer_cli_foreman/role.rb

```diff
@@ -71,6 +71,23 @@ class UpdateCommand < HammerCLIForeman::UpdateCommand
       build_options
     end
 
+    class CloneCommand < HammerCLIForeman::UpdateCommand
+      action :clone
+
+      def self.command_name(name = "clone")
```

Comment:
In this case, it can be shortened to just `command_name "clone"`. But this works too plus it can be overridden from subclasses so NP.